meson.build

```diff
@@ -162,8 +190,25 @@ threadlib = dependency('threads', required: true)
 # Optional dependency on GObject
 build_gobject = false
 if get_option('enable-gobject-types')
-  gobject = dependency('gobject-2.0', version: '>= 2.30.0', required: false)
-  build_gobject = gobject.found()
+  # MSVC builds typically do not use pkg-config, so find GObject/GLib manually
```

Comment:
Hmm, I see...

This follows along the Visual Studio build files we currently have in the GTK+ stack, as no pkg-config is used in those, except when doing introspection builds, which this patch does not cover (yet), as the introspection scanner will make use of pkg-config.